release_layer.py

```diff
@@ -0,0 +1,175 @@
+import codecs
+import re
+import subprocess
+import tempfile
+import shutil
+import os
+import json
+
+# The following values are used in the documentation, so any change of them requires updates to the documentation.
+LAYER_NAME = 'AWSCodeGuruProfilerPythonAgentLambdaLayerTestDev'
```

Comment:
why does this have "TestDev" suffix, why not the official name?

Reply:
Totally by mistake, I used it for testing and forgot it there, I will update it, thanks!